binding.gyp

```diff
@@ -14,6 +14,7 @@
           'libraries': [
             '<(PRODUCT_DIR)/../../windows/lib/libzmq',
             'ws2_32.lib',
+            'iphlpapi'
```

Comment:
Given I have no background on this, it looks like someone fell asleep on the keyboard to name this.